packages/ds-healthcare-gov/src/components/Footer/PrivacySettingsLink.test.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PrivacySettingsLink from './PrivacySettingsLink';
+import { render } from '@testing-library/react';
+
+describe('PrivacySettingsLink', function () {
+  it('should render with default options', () => {
+    const { container } = render(<PrivacySettingsLink />);
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should render with custom class and children', () => {
+    const { container } = render(
+      <PrivacySettingsLink className="ds-t-test-class">Custom modal trigger</PrivacySettingsLink>
+    );
+    expect(container).toMatchSnapshot();
+  });
```

Comment:
The default case is maybe a bit redundant since it's covered in the `Footer` snapshots, but since the custom class and children is specific to usage outside the footer I figured the component needed it's own test file.